frontend/src/components/dashboard/DashboardHeader.tsx

```diff
@@ -107,11 +107,11 @@ export default class DashboardHeader extends React.Component<State> {
             <Link href="/nodes">
               <a>
                 <CardCell
-                  title="Nodes Online"
+                  title="Validators/ Online Nodes"
```

Comment:
Meh, it broke the table since the name is too long. Let's keep "Nodes Online" and later add the (i) helper button (#237) with a popup explaining the numbers.